<?xml version="1.0" encoding="UTF-8"?>
<!--
 Copyright (C) 2018 Dzmitry Hatouka (Gotowka). htotatut@gmail.com

 This program is free software; you can redistribute it and/or modify
 it under the terms of the GNU General Public License as published by
 the Free Software Foundation; either version 3, or (at your option)
 any later version.

 This program is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.

 You should have received a copy of the GNU General Public License
 along with this program. See COPYING.txt
 If not, see <https://www.gnu.org/licenses/>.

 Please note that this program has CLA (Contributor Licence Agreement).
 You accept it by making any contribution to this program.
 See file CONTRIBUTE.txt for more details.
 List of 3-rd party components and their licenses is in 3RDPARTIES.txt
-->
<ui version="4.0">
 <class>Admixture</class>
 <widget class="QDialog" name="Admixture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QGroupBox" name="gbCalculator">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Calculator</string>
   </property>
   <widget class="QComboBox" name="cbCalculator">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>20</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalculate">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>50</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>80</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbChart">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>601</width>
     <height>601</height>
    </rect>
   </property>
   <property name="title">
    <string>Chart</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>581</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>221</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Percentage</string>
   </property>
   <widget class="QTableWidget" name="twPercent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
